test/tests.py

```diff
@@ -1203,7 +1203,8 @@ def test(self):
             join(testdir, '1.2', 'misc', 'rel', 'a', 'x.h'),
             join(testdir, '1.2', 'misc', 'rel', 'a', 'y.h')]
         for exp in expected_subset:
-            if all(os.path.normpath(p) != os.path.normpath(exp)
+            if all(os.path.normpath(os.path.realpath(p))
+                   != os.path.normpath(os.path.realpath(exp))
```

Comment:
Is `.normpath` even needed with `.realpath`?
Also, we're moving away `os.path.realpath` to `Path(p).resolve()`. If realpath is sufficient, please use `Path(p).resolve()` instead.

Reply:
Honestly, I do not know if `normpath` is still needed. I kept it in a "better safe than sorry" mindset. I'll test with `Path(p).resolve()`

Reply:
Changed. Test succeeds.